app/base/models.py

```diff
@@ -23,20 +23,30 @@ class User(db.Model, UserMixin):
     def json(self):
         return {
             "id": self.id,
-            "username": self.name,
+            "username": self.username,
             "email": self.email,
             "profile_image": self.profile_image,
         }
 
     @staticmethod
     def get_all_users():
-        return [User.json(user) for user in User.query.all]
+        return [User.json(user) for user in User.query.all()]
 
     @staticmethod
     def get_user(_user_id):
         query = User.query.get(_user_id)
         return query
 
+    @staticmethod
+    def username_exists(_username):
+        return_value = User.query.filter_by(username=_username).first()
+        return bool(return_value)
```

Comment:
perhaps something  like 
```
    @staticmethod
    def username_exists(_username):
        return bool(User.query.filter_by(username=_username).first().exist())
```

Reply:
I couldn't get it to work. I got `AttributeError: 'NoneType' object has no attribute 'exist'` so I just 
add the code on one line which is converted to a boolean type.

Reply:
😛  that was just expressing an idea not really that you should add `.exist()` method (I meant anything like that)

Reply:
😛 my typo, I forgot to remove `.exist()`